### DISCUSSION TITLE
Quick question

### DISCUSSION THREAD

**Discussion by Ghostscypher** [spatie/laravel-multitenancy#325] [General]
@freekmurze @masterix21 which application or package do you use to convert your markdown files to HTML documentations? I'd like to create a documentation for a small project of mine. I noticed beyondco.de also uses the same documentation format, I'd appreciate a pointer in the right direction.

**Comment by masterix21** [comment]
Hi @Ghostscypher, I'm not in Spatie. Only @freekmurze could reply to you, but I'm sure which documentation works with code from an open-source repository: search in their repositories on GitHub.

**Comment by freekmurze** [comment]
Here you go: https://freek.dev/1795-selling-digital-products-using-laravel-part-7-importing-package-documentation-from-github

**Reply by Ghostscypher** [reply]
Thank you very much @freekmurze and @masterix21 I really appreciate the help. Hoping to be able to give back to the community to like you guys. Thanks a lot. ❤️
